src/PowerShellEditorServices/Services/Workspace/WorkspaceService.cs

```diff
@@ -509,27 +482,6 @@ internal static bool IsPathInMemory(string filePath)
             return isInMemory;
         }
 
-        private string GetBaseFilePath(string filePath)
-        {
-            if (IsPathInMemory(filePath))
-            {
-                // If the file is in memory, use the workspace path
-                return this.WorkspacePath;
-            }
-
-            if (!Path.IsPathRooted(filePath))
-            {
-                // TODO: Assert instead?
-                throw new InvalidOperationException(
```

Comment:
This can never happen because the client only ever sends AbsoluteUris

Reply:
Technically a client that isn't VSCode might do this (just realised this now)